pallets/identity/src/benchmarking.rs

```diff
@@ -1,16 +1,43 @@
 use super::*;
 
 #[allow(unused)]
-use crate::Pallet as Template;
-use frame_benchmarking::{benchmarks, impl_benchmark_test_suite, whitelisted_caller};
+use crate::Pallet as Identity;
+use frame_benchmarking::{account as benchmark_account, benchmarks, impl_benchmark_test_suite, whitelisted_caller};
 use frame_system::RawOrigin;
+use frame_support::inherent::Vec;
+use codec::alloc::string::{ToString, String};
+
+pub fn from_str_to_vec(string: String) -> Vec<u8> {
+    string.as_bytes().to_vec()
+}
+
+pub fn get_account<T: Config>(name: &'static str) -> T::AccountId {
+    let account: T::AccountId = benchmark_account(name, 0, 0);
+    account
+}
+
+pub fn get_origin<T: Config>(name: &'static str) -> RawOrigin<T::AccountId> {
+    RawOrigin::Signed(get_account::<T>(name))
+}
 
 benchmarks! {
-	create_identity {
-		let s in 0 .. 100;
-		let caller: T::AccountId = whitelisted_caller();
-	}: _(RawOrigin::Signed(caller))
-	verify {
-		assert_eq!(IdentityNumber::<T>::get(), s);
-	}
+    create_identity {
+        let s in 0 .. 100;
```

Comment:
Is `s` being used here?

Reply:
good point. I will remove those lines unless I can find a use for them after doing more studying about benchmarks.

Reply:
From what I can see, in the examples they're using them as counters in the verify step. We're kind of offloading the verify off to unit tests right now, though if you can lock down how they actually work that'd be stellar.